kbncontent.go

```diff
@@ -0,0 +1,103 @@
+// Package kbncontent implements routines for analyzing Kibana content.
+//
+// It provides information about Kibana assets and a single source of truth for what things are legacy and should no longer be used.
+package kbncontent
+
+import (
+	"encoding/json"
+	"github.com/PaesslerAG/jsonpath"
```

Comment:
Nit. Imports are usually separated on different blocks separated by empty lines, first standard library packages, then external packages and then internal ones.
```suggestion
	"encoding/json"

	"github.com/PaesslerAG/jsonpath"
```